### PR TITLE
Adding additional root keys

### PR DESCRIPTION
**Pull request by hnykda**
Hi. Mypy was complaining when I tried to add security to the openapi scheme. So I added here.

Taken from https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.2.md#fixed-fields

### CODE REVIEW

File: rest_framework-stubs/schemas/openapi.pyi [hunk 1]
```diff
@@ -17,11 +17,19 @@ class DRFOpenAPIInfo(TypedDict, total=False):
     version: str
     description: str
 
+class externalDocumentationObject(TypedDict):
```
Comment (sobolevn) [inline rest_framework-stubs/schemas/openapi.pyi:20]: `class ExternalDocumentationObject`

**Review by sobolevn — COMMENTED** [review]
Thanks! Is it a `total=True` or `total=False` typed dict?

File: rest_framework-stubs/schemas/openapi.pyi (hunk 1 above)
Comment (hnykda) [inline rest_framework-stubs/schemas/openapi.pyi:20]:
```suggestion
class externalDocumentationObject(TypedDict, total=False):
```

Comment (hnykda) [reply rest_framework-stubs/schemas/openapi.pyi]: False. Fixed!

Comment (hnykda) [reply rest_framework-stubs/schemas/openapi.pyi]: `description` can be missing: 
image

File: rest_framework-stubs/schemas/openapi.pyi [hunk 2]
```diff
@@ -17,11 +17,19 @@ class DRFOpenAPIInfo(TypedDict, total=False):
     version: str
     description: str
 
+class externalDocumentationObject(TypedDict, total=False):
```
Comment (sobolevn) [inline rest_framework-stubs/schemas/openapi.pyi:20]:
```suggestion
class ExternalDocumentationObject(TypedDict, total=False):
```

**Review by sobolevn — CHANGES_REQUESTED** [review]

File: rest_framework-stubs/schemas/openapi.pyi [hunk 3]
```diff
@@ -17,11 +17,19 @@ class DRFOpenAPIInfo(TypedDict, total=False):
...
+    url: str
+
 class DRFOpenAPISchema(TypedDict, total=False):
     openapi: str
     info: DRFOpenAPIInfo
     paths: Dict[str, Dict[str, Any]]
     components: Dict[str, Dict[str, Any]]
+    security: List[Dict[str, List[Any]]]
+    tags: List[Dict[str, Any]]
+    externalDocs: externalDocumentationObject
```
Comment (hnykda) [inline rest_framework-stubs/schemas/openapi.pyi:31]:
```suggestion
    externalDocs: ExternalDocumentationObject
```

**Review by sobolevn — APPROVED** [review]

**Pull request merged**